SQL.script

```diff
@@ -1,4 +1,118 @@
-Create Database SocialMedia
```

Comment:
You can always highlight a part of query and then execute that only in SQL Management Studio